tests/test_profiler.py

```diff
@@ -1,55 +1,96 @@
 import time
-
 import numpy as np
+import pytest
 
 from pytorch_lightning.profiler import Profiler, AdvancedProfiler
 
+PROFILER_OVERHEAD_MAX_TOLERANCE = 0.0001
+
+
+@pytest.fixture
```

Comment:
just to clarify, this freeze the profiler object so all test are using the same instance...

Reply:
it depends on the scope of your fixture. by default, a new object is created for each **function**

https://docs.pytest.org/en/latest/reference.html#pytest-fixture